src/azul/__init__.py

```diff
@@ -1652,9 +1652,13 @@ def google_oauth2_client_id(self) -> str | None:
     def monitoring_email(self) -> str:
         return self.environ['AZUL_MONITORING_EMAIL']
 
-    @property
-    def cloudwatch_dashboard_template(self) -> str:
-        return f'{config.project_root}/terraform/cloudwatch_dashboard.template.json.py'
+    def cloudwatch_dashboard_template(self, name: str) -> str:
+        valid_names = ['indexer']
+        if config.enable_mirroring:
+            valid_names.append('mirror')
```

Comment:
Honestly not sure if this is an improvement, but you could also use a more declarative style here:
```suggestion
        valid_names = ['indexer', *iif(config.enable_mirroring, ['mirror'])]
```